pkg/model/components/kubeapiserver/model.go

```diff
@@ -126,7 +127,7 @@ spec:
 func (b *KubeApiserverBuilder) buildHealthcheckSidecar() (*corev1.Pod, error) {
 	// TODO: pull from bundle
 	bundle := "(embedded kube-apiserver-healthcheck manifest)"
-	manifest := []byte(defaultManifest)
+	manifest := []byte(fmt.Sprintf(defaultManifest, wellknownports.KubeAPIServerHealthCheck))
```

Comment:
Nit: I normally try to avoid using Sprintf with our templates, favoring `{{ HealthCheckPort }}` and `strings.ReplaceAll`,  because the `%d` is harder to read, and has more gotchas (repeated placeholders don't work, it's vaguely possible that a `%` placeholder could appear 'accidentally')